core/include/kernel/interrupt.h

```diff
@@ -28,6 +34,15 @@ struct itr_chip {
 			  uint32_t *prio);
 };
 
+/*
+ * struct itr_ops - Interrupt controller operations
+ * @add		Register and configure a interrupt line
```

Comment:
The word "line" used here and in the following lines is redundant, please remove it.
s/a interrupt/an interrupt/